ts/features/bonus/bonusVacanze/components/AvailableBonusItem.tsx

```diff
@@ -93,13 +89,11 @@ export const AvailableBonusItem: React.FunctionComponent<Props> = (
               )}
             </View>
           </Row>
-          {maybeSponsorDescription.isSome() && (
-            <Row>
-              <Text style={[styles.servicesName, disabledStyle]}>
-                {maybeSponsorDescription.value}
-              </Text>
-            </Row>
-          )}
+          <Row>
+            <Text style={[styles.servicesName, disabledStyle]}>
+              {bonusTypeLocalizedContent.description}
```

Comment:
```suggestion
              {bonusTypeLocalizedContent.description ?? ""}
```
Since it could be undefined, may it be an improvement?